readme.md

```diff
@@ -39,6 +43,16 @@ Default: `process.cwd()`
 
 Path to the directory you want to check.
 
+#### options
+
+Type: `object`
+
+##### files
+
+Type: `array`
+
+An array of [`multimatch`](https://github.com/sindresorhus/multimatch) patterns to ignore certain files.
+
 ### isGitClean.sync([dir])
```

Comment:
I think we should just get rid of the duplicate documentation for the `sync` version, instead it should just read:

> Synchronous version, with the same API as above. Returns a `boolean` directly, instead of a Promise.